asyncpgsa/connection.py

```diff
@@ -15,32 +15,41 @@
 _dialect.paramstyle = 'named'
 
 
+def replace_keys(keys, querystring, params, inline=False):
+    new_query = querystring
+    new_params = []
+    for key, value in enumerate(keys):
+        if inline:
+            new_query = new_query.replace(':' + value, params[key][1], 1)
```

Comment:
this has the same faulty logic as before. 
If I have `int_10` and it comes before `int_1`, than `int_10` will be replace by the `int_1`. Also this fails if the variable exists more than once in the string.
In order to solve this issue, we need to use regex to do the replacement, not just the finding.